linux/ibus-keyman/meson.build

```diff
@@ -9,10 +9,10 @@ project('ibus-keyman', 'c', 'cpp',
 cc = meson.get_compiler('c')
 conf = configuration_data()
 
-ibus =        dependency('ibus-1.0',             version: '>= 1.2.0')
-gtk =         dependency('gtk+-3.0',             version: '>= 2.4')
-x11 =         dependency('x11',                  version: '>= 1.6')
-json_glib =   dependency('json-glib-1.0',        version: '>= 1.0')
+ibus =           dependency('ibus-1.0',             version: '>= 1.2.0')
+gtk =            dependency('gtk+-3.0',             version: '>= 2.4')
+json_glib =      dependency('json-glib-1.0',        version: '>= 1.0')
```

Comment:
Is x11 still used on older distros?

Reply:
Yes, X11 can still be used, even on newer distros, but this code no longer has a dependency on it (turns out that the way I implemented capslock handling for Wayland also works on X11 and so we no longer need the distinction between X11 and Wayland. In fact, the new code works better - previously when we programmatically turned caps lock off the caps lock indicator was still on until after the next key press. With the new code it immediately turns off).